test/integration/popover-tooltip.test.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from '@vaadin/chai-plugins';
+import { aTimeout, fixtureSync, nextRender } from '@vaadin/testing-helpers';
+import './not-animated-styles.js';
+import '@vaadin/popover/src/vaadin-popover.js';
+import '@vaadin/tooltip/src/vaadin-tooltip.js';
+import { mouseleave } from '@vaadin/popover/test/helpers';
+import { mouseenter } from '@vaadin/tooltip/test/helpers';
```

Comment:
Actual helper files seem identical so we can import from a single package:

```suggestion
import { mouseenter, mouseleave } from '@vaadin/popover/test/helpers.js';
```

Reply:
Done